app/components/generic/select/FormMultiSelect.tsx

```diff
@@ -38,7 +38,7 @@ export default function FormMultiSelect({
   return (
     <div className={classNames?.wrapper}>
       {label && (
-        <Label htmlFor={id} className={classNames?.label}>
+        <Label htmlFor={id} className={classnames(classNames?.label, 'mb-2')}>
```

Comment:
please leave the default margin and override it from the end usage by passing `classNames.label`.